### PR TITLE
Update svm_author_id.py

### PR DESCRIPTION
**Pull request by pps121**
completed the TODO part of the SVM code.

### CODE REVIEW

**Comment by pps121** [comment]
from sklearn import svm
clf = svm.SVC(kernel="rbf")
clf.fit(features_train, labels_train)
#Hyper-marameters are 'C', 'gamma', and the 'kernel'  
SVC(C=1.0, cache_size=200, class_weight=None, coef0=0.0,
    decision_function_shape='ovr', degree=3, gamma='scale', kernel='rbf',
    max_iter=-1, probability=False, random_state=None, shrinking=True,
    tol=0.001, verbose=False)
pred=clf.predict(features_test) # predict with test_features

#print accuracy using the classifier
from sklearn.metrics import accuracy_score
print(accuracy_score(pred, labels_test) # i.e. predicted-label VS true-label

File: svm/svm_author_id.py [hunk 1]
```diff
@@ -24,7 +24,20 @@
...
+SVC(C=1.0, cache_size=200, class_weight=None, coef0=0.0,
+    decision_function_shape='ovr', degree=3, gamma='scale', kernel='rbf',
+    max_iter=-1, probability=False, random_state=None, shrinking=True,
+    tol=0.001, verbose=False)
+pred=clf.predict(features_test) # predict with test_features
+
+#print accuracy using the classifier
+from sklearn.metrics import accuracy_score
+print(accuracy_score(pred, labels_test) # i.e. predicted-label VS true-label
+
```
Comment (pps121) [inline svm/svm_author_id.py]: Completed the TODO part of SVM code. Please check and commit.
